remote-scripts/stor_add_disk_ide.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+###############################################################################
+##
+## Description:
+##   add IDE disk and check wheather it works
+##
+###############################################################################
+##
+## Revision:
+## v1.0.0 - ruqin - 7/11/2018 - Build the script
+##
+###############################################################################
+
+dos2unix utils.sh
+
+#
+# Source utils.sh
+#
+. utils.sh || {
+    echo "Error: unable to source utils.sh!"
+    exit 1
+}
+
+#
+# Source constants file and initialize most common variables
+#
+UtilsInit
+
+###############################################################################
+##
+## Put your test script here
+## NOTES:
+## 1. Please use LogMsg to output log to terminal.
+## 2. Please use UpdateSummary to output log to summary.log file.
+## 3. Please use SetTestStateFailed, SetTestStateAborted, SetTestStateCompleted,
+##    and SetTestStateRunning to mark test status.
+##
+###############################################################################
+
+
+# Do Partition for /dev/sdb
```

Comment:
1. Where is the operation for Add a IDE?

Reply:
add ide is the test case name